src/reducers/userReducer.js

```diff
@@ -0,0 +1,24 @@
+import { userActionTypesConstants } from "constants/userConstants";
+
+const initialState = {
+  loading: false,
+};
```

Comment:
Consider add more data to this user state. It doesn't have too much sense add to redux store only a simple variable to say if it's loading or not

Reply:
I do this to show a loader